src/drivers/indexeddb.js

```diff
@@ -181,27 +181,164 @@
             }
         }
 
+        // Initialize a singleton container for all running localForages.
+        if (!globalObject.container) {
+            globalObject.container = {};
```

Comment:
Is there a reason that we attach the `container` on the global object?? It feels too intrusive to me to just add an extra variable to the window object.

This could probably be just a local variable [around line 18](https://github.com/sebweaver/localForage/blob/issue-342-multiple-stores-on-same-db/src/drivers/indexeddb.js#L18), inside the closure of the IndexedDB driver.

As an alternative, it could be attached on the LocalForage class as a static property, named like `_dbContexts`. On the other hand I don't see a reason that this couldn't just be completely private to the IndexedDB driver.